charts/enterprise-metrics/templates/_helpers.tpl

```diff
@@ -0,0 +1,61 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "metrics_enterprise.name" -}}
```

Comment:
Helper templates should use the name of the chart.
```suggestion
{{- define "enterprise-metrics.name" -}}
```

Reply:
Good catch, I missed this during find and replace, thanks!